content/document.js

```diff
@@ -42,6 +42,28 @@ function updateWikiHistory(localeContentRoot, oldSlug, newSlug = null) {
   }
 }
 
+function updateDeletedFile(contentRoot, locale, slug) {
+  let deletedLines = "";
+  const deletedFile = path.join(
+    contentRoot,
+    locale.toLowerCase(),
+    "_deleted.txt"
+  );
+  if (fs.existsSync(deletedFile)) {
+    deletedLines = fs.readFileSync(deletedFile, "utf-8");
+  } else {
+    deletedLines =
+      "# This file is generated when you use the CLI tool " +
+      "to delete slugs.\n\n";
+  }
+  const url = buildURL(locale, slug);
+  const newLine = `${url.padEnd(60, " ")} # ${new Date().toDateString()}\n`;
```

Comment:
```suggestion
  const newLine = `${url.padEnd(60, " ")}\t# ${new Date().toDateString()}\n`;
```
can we use a tab here? Easier to split and process.